packages/website/ts/components/relayer_index/relayer_index.tsx

```diff
@@ -59,10 +59,11 @@ export class RelayerIndex extends React.Component<RelayerIndexProps, RelayerInde
         const readyToRender = _.isUndefined(this.state.error) && !_.isUndefined(this.state.relayerInfos);
         if (!readyToRender) {
             return (
-                <div className="col col-12" style={{ ...styles.root, height: '100%' }}>
+                // TODO: consolidate this loading component with the one in portal
+                <div className="pt4 sm-px2 sm-pt2 sm-m1" style={{ height: 500 }}>
                     <div
                         className="relative sm-px2 sm-pt2 sm-m1"
-                        style={{ height: 122, top: '33%', transform: 'translateY(-50%)' }}
+                        style={{ height: 122, top: '50%', transform: 'translateY(-50%)' }}
```

Comment:
Is this to center it within the parent or something? We can use flexbox right (don't need to support older browsers)?

Reply:
Yep, we use flexbox.